electron/package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@bitwarden/jslib-common": "file:../common",
     "@nodert-win10-rs4/windows.security.credentials.ui": "^0.4.4",
-    "electron": "14.0.1",
+    "electron": "^14.1.1",
```

Comment:
We usually don't want `^` for this since it might accidentally update itself in the future.

Reply:
Discussed with Oscar to update to `14.2.0` as that is the latest for `v14` and removed the `^` t fix it to that version.